README.md

```diff
@@ -1,159 +1,170 @@
+#### Ingesting the Data
 
-To ingest and curate (harmonize) the sample data provided, run the following commands at the top level of the ../healthcare-starter-kit directory into which you deployed the project artifacts:
+To ingest all data you can run `./gradlew ingest`, or to ingest a smaller set of claims (for faster setup) you can run `./gradlew ingestSmaller`.
 
-#### Ingest the Data
+If you would like to load sets of data individually you can run the tasks that the above depend on instead:
 
-Run the following commands in sequence for ingestion of the provided sample data.
- 
+```sh
+./gradlew ingestClaimLines # or ingestClaimLinesSmaller
+./gradlew ingestClaims # or ingestClaimsSmaller
+./gradlew ingestOrganizations
+./gradlew ingestPatients
+./gradlew ingestPayers
+./gradlew ingestProviders
 ```
-# Patient (755 documents)
-./gradlew hubRunFlow -PentityName=Patient -Psteps="1" -PflowName=Patient -PinputFilePath=data/synthea/csv/patients -PenvironmentName=local
 
-# Provider (790 documents)
-./gradlew hubRunFlow -PentityName=Provider -Psteps="1" -PflowName=Provider -PinputFilePath=data/synthea/csv/providers -PenvironmentName=local
-
-# Organization (790 documents)
-./gradlew hubRunFlow -PentityName=Organization -Psteps="1" -PflowName=Organization -PinputFilePath=data/synthea/csv/organizations -PenvironmentName=local
-
-# Claim (76,118 documents)
-./gradlew hubRunFlow -PentityName=Claim -Psteps="1" -PflowName=Claim -PinputFilePath=data/synthea/csv/claims -PenvironmentName=local
-
-# ClaimTransaction (464,726 documents)
-./gradlew hubRunFlow -PentityName=ClaimTransaction -Psteps="1" -PflowName=ClaimTransaction -PinputFilePath=data/synthea/csv/claims_transactions -PenvironmentName=local
-
-# Payor (10 documents)
-./gradlew hubRunFlow -PentityName=Payor -Psteps="1" -PflowName=Payor -PinputFilePath=data/synthea/csv/payers -PenvironmentName=local
-```
+#### Curating the Data
 
-#### Curate the Data
+To curate all previously ingested data you can run `./gradlew harmonizeAll`.
 
-Run the following commands in sequence for curation (mapping, custom, matching, and merging steps) of the provided sample data.
+If you would like to curate sets of data individually you can run the tasks that the above depends on instead:
 
-```
-# Patient
-./gradlew hubRunFlow -PentityName=Patient -Psteps="2,3,4,5" -PflowName=Patient -PenvironmentName=local
-
-# Organization
-./gradlew hubRunFlow -PentityName=Organization -Psteps="2,3" -PflowName=Organization -PenvironmentName=local
-
-# Provider
-./gradlew hubRunFlow -PentityName=Provider -Psteps="2,3,4" -PflowName=Provider -PenvironmentName=local
-
-# Claim
-./gradlew hubRunFlow -PentityName=Claim -Psteps="2" -PflowName=Claim -PenvironmentName=local
+```sh
+./gradlew harmonizeClaims
+./gradlew harmonizeOrganizations
+./gradlew harmonizePatients
+./gradlew harmonizeProviders
 ```
 
 ### Running Unit and Integration Tests
 
 To verify the deployment, two test suites are provided.

```

Comment:
Add a clarification that the ClaimsSuite is a stand-alone test suite but other tests run against the curated sample data so the ingest and harmonize tasks must be run first.